File: 15 exercicios POO/1/golang/main.go
package main

import "fmt"

// Definindo a struct Carro
type Carro struct {
	Marca  string
	Modelo string
	Ano    int
}

func main() {
	// Criando uma instância de Carro
	carrinho := Carro{
		Marca:  "Bonito",
		Modelo: "Grande",
		Ano:    166,
	}

	// Imprimindo os atributos do carro
	fmt.Println("Marca do veículo:", carrinho.Marca)
	fmt.Println("Modelo do veículo:", carrinho.Modelo)
	fmt.Println("Ano do veículo:", carrinho.Ano)
}

File: 15 exercicios POO/1/java/src/Main.java
public class Main {
    public static void main(String[] args) {
        Carro carrinho = new Carro();
        carrinho.marca = "Bonito";
        carrinho.modelo = "Grande";
        carrinho.ano = "1666";

        System.out.println(
                "Marca do veículo: "+carrinho.marca+
                "\nModelo do veículo: "+carrinho.modelo+
                "\nAno do veículo: "+carrinho.ano
        );
    }
}